skimage/restoration/tests/test_restoration.py

```diff
@@ -1,15 +1,22 @@
 from os.path import abspath, dirname, join as pjoin
+from pathlib import Path
```

Comment:
I've probably missed a trend... Is `os.path.join` too mainstream now?

Reply:
I decided I want to backport this, and it might be better to just use os.path.join for an other year.